### DISCUSSION TITLE
Keep meshes separate (don't combine layers)?

### DISCUSSION THREAD

**Discussion by bigrig2212** [donmccurdy/glTF-Transform#850] [Q&A] [answered]
Hi there - wondering if there's an option to keep layers as is in the mesh, versus combining them all into a single layer. 

i was trying:
`gltf-transform optimize "$file" "output/$filename" --join false --simplify 0.0`

and 
`gltf-transform optimize "$file" "output/$filename" --simplify 0`

But those didn't work. Suppose im just not finding the right option?

Eg: Maintain this structure
<img width="294" alt="Screenshot 2023-03-10 at 11 03 54 PM" src="https://user-images.githubusercontent.com/19869387/224470371-e59403c0-6ce5-4491-8098-339fabd840a9.png">

Versus combine it into this:
<img width="286" alt="Screenshot 2023-03-10 at 11 04 05 PM" src="https://user-images.githubusercontent.com/19869387/224470387-f0ea7bf8-feda-435e-ad8f-c645bfa45829.png">

**Comment by donmccurdy** [comment, marked as answer]
The `--join false --flatten false` options were added in v3.1, by #838, so you may need to update to the latest version. Then this would work:

```
gltf-transform optimize in.glb out.glb --join false --flatten false --simplify 0.0
```

Or `--no-flatten --no-join` should have the same effect.

**Comment by bigrig2212** [comment]
Wonderful! Works like a charm.
